content/docs/command-reference/fetch.md

```diff
@@ -51,6 +46,15 @@ project's cache                  ++ | dvc pull |
  workspace
 ```
 
+Without arguments, it downloads all files and directories missing from the
+project, found as <abbr>outputs</abbr> in all `dvc.yaml` and `.dvc` files
```

Comment:
Without arguments, it downloads files referenced in the current workspace (reads `.dvc`, `dvc.yaml`, `dvc.lock` to find <abbr>outputs</abbr> and DVC-tracked data). ...

Reply:
Like it. Updated here and in pull, push.